contrib/release

```diff
@@ -118,8 +118,7 @@ fn-version-next() {
   elif [[ "$RELEASE" == "patch" ]]; then
     patch=$((patch + 1))
   else
-    short_object=$(git log --oneline | tail -n 1 | cut -d' ' -f1)
-    echo "${CURRENT_VERSION}build+$(echo "$DOKKU_GIT_REV" | cut -c1-${#short_object})"
+    echo "$(git describe --tags | cut -c2-)"
```

Comment:
This is necessary for us to have build releases. I would revert.

Reply:
`git describe --tags` gives output like `v0.12.13-46-ge7f7cb8` so that doesn't prevent the build releases. Does the word _build_ matter in this, or was it just the existing convention?

Reply:
Yeah, it kinda does. Deb packaging is weird.

Reply:
reverted